test/test/lock/LockTests.java

```diff
@@ -22,15 +23,15 @@ private static void contendedLocks(TestProcess p, int interval, double minRatio)
     }
 
     // 0 is equivalent to disabling sampling of locks, so all profiles are included.
-    @Test(mainClass = LockProfiling.class, inputs = { "0", "70" })
-    @Test(mainClass = LockProfiling.class, inputs = { "10000", "70" })
+    @Test(mainClass = LockProfiling.class, inputs = { "0", "70" }, os = Os.LINUX)
+    @Test(mainClass = LockProfiling.class, inputs = { "10000", "70" }, os = Os.LINUX)
 
     // Large (for the specific paylod) interval value skews the sampled lock
     // contention distribution.
-    @Test(mainClass = LockProfiling.class, inputs = { "1000000", "90" })
+    @Test(mainClass = LockProfiling.class, inputs = { "1000000", "90" }, os = Os.LINUX)
```

Comment:
Lock tests sometimes fail on Linux, too.
Let's disable them for now and ask @krk to amend.

Reply:
Will disable the tests in another PR